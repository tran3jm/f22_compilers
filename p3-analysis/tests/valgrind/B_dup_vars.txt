==1130947== Memcheck, a memory error detector
==1130947== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1130947== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==1130947== Command: ../decaf inputs/dup_var.decaf
==1130947== 
Duplicate symbols named 'a' scope on line 1
==1130947== 
==1130947== HEAP SUMMARY:
==1130947==     in use at exit: 0 bytes in 0 blocks
==1130947==   total heap usage: 1,775 allocs, 1,775 frees, 214,560 bytes allocated
==1130947== 
==1130947== All heap blocks were freed -- no leaks are possible
==1130947== 
==1130947== For lists of detected and suppressed errors, rerun with: -s
==1130947== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
