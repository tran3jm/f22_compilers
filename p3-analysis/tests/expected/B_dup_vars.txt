Duplicate symbols named 'a' scope on line 1
